Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs

```diff
@@ -120,6 +120,23 @@ public QuestionController(IQuestionRepository questionsRepository, UserManager<A
 
             return Ok(questionAC);
         }
+
+        /// <summary>
+        /// Returns Question of specific Id
+        /// </summary>
+        /// <param name="id">Id of Question</param>
+        /// <returns>
+        /// QuestionAC class object
+        /// </returns>
+        [HttpGet("{id}")]
+        public async Task<IActionResult> GetQuestionByIdAsync([FromRoute]int id)
+        {
+            if (!await _questionsRepository.IsQuestionExistAsync(id))
```

Comment:
Please use GetQuestionById method and if that object do not found return 404 else return object.